doc/manual/command-ref/conf-file.xml

```diff
@@ -430,6 +430,21 @@ flag, e.g. <literal>--option gc-keep-outputs false</literal>.</para>
   </varlistentry>
 
 
+  <varlistentry><term><literal>netrc-file</literal></term>
+
+    <listitem><para>If set to an absolute path to a <filename>netrc</filename>
```

Comment:
@shlevy the closest thing I found to a spec was `netrc(5)`, but that talks about FTP. I inlined an example here because I couldn't find a good reference. The parser used by `curl` isn't more helpful: https://github.com/curl/curl/blob/master/lib/netrc.c